modern-gpus/intel-gpu.md

```diff
@@ -1,8 +1,8 @@
 # Intel GPUs
 
-So I'll be going over the compatible iGPUs present in Intel's CPUs, the main thing to note is that you'll need to apply the FrameBuffer patch to your system to get things to work properly. [Please refer to this post for more info on Framebuffer patching as it goes in depth on how to get your system running](https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.IntelHD.en.md). We will also be excluding iGPUs present in Pentiums, Celerons and Atom CPUs as these generally have never been supported natively and require quite a bit of extra work to get them working(to be more specific, GT1 based iGPUs don't work, Apple only uses GT2 and up in their macs)
+So we'll be going over the compatible iGPUs present in Intel's CPUs. The main thing to note is that you'll need to apply the framebuffer patches to your system to get things to work properly. [Please refer to this post for more info on framebuffer patching as it goes in depth on how to get your system running](https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.IntelHD.en.md). We will also be excluding iGPUs present in Pentiums, Celerons and Atom CPUs as these generally have never been supported natively and require quite a bit of extra work to get them working (to be more specific, GT1 based iGPUs don't work, Apple only uses GT2 and up in their Macs).
 
-**DRM Issues**: With Haswell and newer iGPUs, DRM is outright broken on them with macOS Catalina. This includes iTunes Movies, Apple TV+, Amazon Prime and Netflix, the only fix is getting a supported dGPU preferably Polaris or newer that supports HEVC.
```

Comment:
HEVC is required for some stuff like sidecar, not too sure about DRM.

Reply:
Yeah I don't think HEVC is required for DRM either, GCN 1.0 doesn't support HEVC but people are still able to get DRM working. What would you like me to do here? I removed it because I have a Kaby Lake laptop on macOS 11 and FairPlay 1, 2, and 3 work. Haven't tested FairPlay 4.